bundles/binding/org.openhab.binding.zibase/src/main/java/org/openhab/binding/zibase/internal/ZibaseBindingConfigX10Event.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2014, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zibase.internal;
+
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+
+import fr.zapi.Zibase;
+
+/**
+ * This class handle X10 Event
+ * TODO : Implement
+ * @author Julien Tiphaine
+ *
+ */
+public class ZibaseBindingConfigX10Event extends ZibaseBindingConfig {
+
+	public ZibaseBindingConfigX10Event(String[] configParameters) {
+		super(configParameters);
+		// TODO Auto-generated constructor stub
+	}
+
+	@Override
+	public State getOpenhabStateFromZibaseValue(Zibase zibase, String zbResponseStr) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public void sendCommand(Zibase zibase, Command command, int dim) {
+		// TODO Auto-generated method stub
+		
+	}
+
+	@Override
+	protected boolean isItemConfigValid() {
+		// TODO Auto-generated method stub
+		return false;
+	}
+
+}
```

Comment:
hmm … what is this placeholder class for? Better remove it and readd it once you've implemented it. If not please remove all TODOs and document the class properly.